crates/bevy_ecs/src/storage/table.rs

```diff
@@ -32,28 +33,38 @@ impl TableId {
 
 #[derive(Debug)]
 pub struct Column {
```

Comment:
Why is this pub?

Reply:
Table allows access via `iter` and you can get individual columns and their corresponding component IDs. It does allow direct mutation via pointer/unsafe APIs, but doesn't allow for structural changes without `&mut Column`.

Also until #4809 is merged, this at least needs to remain `pub(crate)`.